test/simulator/main/org/apache/cassandra/simulator/paxos/PaxosSimulation.java

```diff
@@ -286,19 +288,39 @@ RuntimeException failWith(Throwable t)
 
     private RuntimeException logAndThrow()
```

Comment:
This doesn't return a RuntimeException, but I know it can also be handy for this kind of method to return something to throw so in the calling method the compiler knows execution stops there.